Data-Analytics/Spark/DataProcessTransfer/DataProcessTransferFts.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: "sparkoperator.hpe.com/v1beta2"
+kind: SparkApplication
+metadata:
+  name: data-process-transfer-demo
+  namespace: <username>
+spec:
+  sparkConf:
+    spark.hadoop.fs.s3a.path.style.access: "true"
+    spark.hadoop.fs.s3a.impl: "org.apache.hadoop.fs.s3a.S3AFileSystem"
+    spark.hadoop.fs.s3a.connection.ssl.enabled: "false"
+    spark.mapr.extraconf.secret: "<k8s-secret-name>"
```

Comment:
Same here. To access local-s3-service we can switch to using EzSparkAWSCredentialProvider instead of having custom secret for that.